tests/test_losses.py

```diff
@@ -34,3 +34,14 @@ def test_no_bayes(self):
 
         loss = ELBOLoss(model, criterion, kl_weight=1e-5, num_samples=1)
         loss(model(torch.randn(1, 1)), torch.randn(1, 1))
+
+
+# fmt: on
+class TestNIGLoss:
+    def test_main(self):
+        with pytest.raises(ValueError):
+            NIGLoss(reg_weight=-1)
+
+        loss = NIGLoss(reg_weight=1e-2)
+
+        loss(*torch.rand((1, 4)).split(1, dim=-1), torch.randn(1, 1))
```

Comment:
Perhaps include one assert based on an expected result?

Reply:
I've added the expected outcome when all four parameters and the target are set to 1.